src/schema/flight.schema.ts

```diff
@@ -28,7 +28,7 @@ export type DistanceUnit = z.TypeOf<typeof distanceUnitSchema>;
 export type UnregisteredFlightRequest = z.TypeOf<
   typeof unregisteredFlightRequestSchema
 >;
-
+/* Fix typo */
 const flightResponseUnqiue = z.object({
```

Comment:
You can go ahead and fix it while you're on here. It'll be fine. Just change it in the type def too.

Reply:
Done